zipline-cli/README.md

```diff
@@ -1,6 +1,6 @@
 zipline-cli
```

Comment:
With the `zipline-cli` bash file removed, could the docs be updated for how to use the CLI after these changes?

Reply:
Yeah I was being lazy. It's basically the same, though, but the script is inside the zip now.

Reply:
Or if you're building locally it's exactly the same it's just in a new location.